gnucash/gnome/gnc-plugin-page-account-tree.cpp

```diff
@@ -1595,7 +1592,7 @@ confirm_delete_account (GSimpleAction *simple, GncPluginPageAccountTree *page,
                                 acct_name);
     g_free(acct_name);
 
-    if (splits)
+    if (!splits.empty())
```

Comment:
It looks like this is the only use of `splits`, so lose the decl at line 1581 and say 
`if (!xaccAccountGetSplits(account).empty())` 
here.